modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js

```diff
@@ -442,7 +517,12 @@ class SegmentEdit extends Component {
 											className="text-capitalize"
 											disabled={disabledSaveButton}
 											displayType="primary"
-											onClick={this._handleValidate}
+											onClick={(event) =>
+												this._handleValidate({
+													contributors,
```

Comment:
If you have contributors in the state maybe you don't need to pass ir to handleValidate

Reply:
Absolutely!